doc-content/shared-kie-docs/src/main/asciidoc/KieServer/kie-server-system-properties-ref.adoc

```diff
@@ -0,0 +1,361 @@
+
+|`org.jbpm.task.cleanup.enabled`
+|`true`, `false`
+|`true`
+|Enables task cleanup job listener to remove tasks once the process instance is completed.
+
+|`org.jbpm.task.bam.enabled`
+|`true`, `false`
+|`true`
+|Enables task BAM module to store task related information.
+
+|`org.jbpm.ht.admin.user`
+|String
+|`Administrator`
+|User who can access all the tasks from {KIE_SERVER}.
+
+|`org.jbpm.ht.admin.group`
+|String
+|`Administrators`
+|In order to display all the tasks from {KIE_SERVER}, users must belong to the `Administrators` group.
+|===
+endif::[]
+
+.System properties for loading keystore
+[cols="2,1,1,3", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`kie.keystore.keyStoreURL`
+|URL
+|N/A
+|URL is used to load a Java Cryptography Extension KeyStore (JCEKS). For example, `file:///home/kie/keystores/keystore.jceks`.
+
+|`kie.keystore.keyStorePwd`
+|String
+|N/A
+|The password is used for the JCEKS.
+
+|`kie.keystore.key.server.alias`
+|String
+|N/A
+|The alias name of the key for REST services where the password is stored.
+
+|`kie.keystore.key.server.pwd`
+|String
+|N/A
+|The password of an alias for REST services.
+
+|`kie.keystore.key.ctrl.alias`
+|String
+|N/A
+|The alias of the key for default REST {CONTROLLER}.
+
+|`kie.keystore.key.ctrl.pwd`
+|String
+|N/A
+|The password of an alias for default REST {CONTROLLER}.
+|===
+
+.Other system properties
+[cols="2,1,1,3a", options="header"]
+|===
+|Property
+|Values
+|Default
+|Description
+
+|`kie.maven.settings.custom`
+|Path
+|N/A
+|The location of a custom `settings.xml` file for Maven configuration.
+
+|`kie.server.jms.queues.response`
+|String
+|`queue/KIE.SERVER.RESPONSE`
+|The response queue JNDI name for JMS.
+
+|`org.drools.server.filter.classes`
+|`true`, `false`
+|`false`
+|When set to `true`, the Drools {KIE_SERVER} extension accepts custom classes annotated by the `XmlRootElement` or `Remotable` annotations only.
+
+ifdef::PAM,JBPM[]
+|`org.kie.server.bypass.auth.user`
+|`true`, `false`
+|`false`
+|A property that enables you to bypass the authenticated user for task-related operations, for example queries.
+
+|`org.jbpm.rule.task.firelimit`
+|Integer
+|`10000`
+|A property that specifies maximum number of fired rules to avoid situation where rules run into infinite loop and make server completely unresponsive.
+
+|`org.jbpm.ejb.timer.local.cache`
+|`true`, `false`
+|`true`
+|This property turns off the EJB Timers local cache.
+endif::[]
+
+|`org.kie.server.domain`
+|String
+|N/A
+|The JAAS `LoginContext` domain used to authenticate users when using JMS.
+
+|`org.kie.server.repo`
+|Path
+|`.`
+|The location where {KIE_SERVER} state files will be stored.
+
+|`org.kie.server.sync.deploy`
+|`true`, `false`
+|`false`
+|A property that instructs the {KIE_SERVER} to hold the deployment until the {CONTROLLER} provides the containers deployment configuration. This property only affects servers running in managed mode. The options are as follows:
```

Comment:
"provides the container deployment"

Also, can't say "as follows", "the following", etc. per IBM SG. Suggest "The following options are available:"